internal/generator/fluentd/output/syslog/output_conf_syslog_test.go

```diff
@@ -357,6 +355,47 @@ var _ = Describe("Generating external syslog server output store config blocks",
 					Expect(results).To(EqualTrimLines(tcpWithTLSConf))
 				})
 			})
+			Context("with TLS enabled and Hostname Verify", func() {
+				BeforeEach(func() {
+					outputs = []logging.OutputSpec{
+						{
+							Type: "syslog",
+							Name: "syslog-receiver",
+							URL:  "tls://sl.svc.messaging.cluster.local:9654",
+							Secret: &logging.OutputSecretSpec{
+								Name: "some-secret",
+							},
+						},
+					}
+					secret = &corev1.Secret{
+						Data: map[string][]byte{
+							"ca-bundle.crt":          []byte("junk"),
+							"syslog_hostname_verify": []byte("true"),
+						},
+					}
+				})
+				It("should produce security config with host name verification", func() {
+					c := SecurityConfig(outputs[0], secret)
+					results, err := g.GenerateConf(c...)
+					Expect(err).To(BeNil())
+					Expect(results).To(EqualTrimLines(`
+						tls true
+						verify_mode 1 #VERIFY_NONE:0, VERIFY_PEER:1
+						ca_file '/var/run/ocp-collector/secrets/some-secret/ca-bundle.crt'
+					`))
+				})
+				It("should produce security config with host name verification disabled", func() {
+					secret.Data["syslog_hostname_verify"] = []byte("false")
```

Comment:
Can we refer to `const SyslogHostnameVerify` here?

Reply:
done